lib/ts/ink_config.h.in

```diff
@@ -127,5 +127,6 @@
 #define TS_BUILD_CANONICAL_HOST "@host@"
 
 #define TS_BUILD_DEFAULT_LOOPBACK_IFACE "@default_loopback_iface@"
+/* clang-format off */
```

Comment:
Thanks, yeah, should be on (copy-paste failure). So yeah, we don't necessarily need this, however, I ended up screwing this up myself since I edited this file, and wanted to make sure clang-format didn't change anything. And then of course, the entire file gets foobared. I think it's a good measure to prevent zwoop from screwing it up again.